module_utils/network_lsr/argument_validator.py

```diff
@@ -246,6 +247,62 @@ def _validate_string_min_length(self, value):
             return True
 
 
+class ArgValidatorRouteTable(ArgValidator):
+    def __init__(
+        self,
+        name,
+        required=False,
+        default_value=None,
+    ):
+        ArgValidator.__init__(self, name, required, default_value)
+
+    def _validate_impl(self, value, name):
+        norm_value = None
+        try:
+            if isinstance(value, bool):
+                # bool can (probably) be converted to self.numeric_type,
+                # but here we don't want to accept a boolean value.
+                pass
+            elif isinstance(value, int):
+                norm_value = int(value)
+            elif isinstance(value, Util.STRING_TYPE):
+                try:
+                    norm_value = int(value)
+                except Exception:
+                    norm_value = value
+        except Exception:
+            pass
+        if norm_value is None:
+            raise ValidationError(
+                name,
+                "route table must be the named or numeric tables but is '%s'" % (value),
+            )
+        if isinstance(norm_value, int):
+            if norm_value < 1:
+                raise ValidationError(
+                    name,
+                    "route table value is %s but cannot be less then 1" % (value),
+                )
+            elif norm_value > 0xFFFFFFFF:
+                raise ValidationError(
+                    name,
+                    "route table value is %s but cannot be greater then 4294967295"
+                    % (value),
+                )
+        if isinstance(norm_value, Util.STRING_TYPE):
+            if norm_value == "":
+                raise ValidationError(
+                    name,
+                    "route table name can not be empty string" % (value),
```

Comment:
This is using the % operator without a "%s" in the string, the % operator does not seem to be necessary here. Also, the other code is using "cannot" instead of "can not".